style.css

```diff
@@ -33,3 +33,35 @@ a {
 	text-decoration-thickness: .0625em !important;
 	text-underline-offset: .15em;
 }
+
+
+
+/*
+ * Styles for the custom Faq checkmark list block style
+ * https://github.com/WordPress/twentytwentyfour/issues/46
+ */
+ .is-style-faq-check-lists {
+	list-style-position: inside;
+	list-style-type: none;
+	padding-left: 0px;
+}
+
+.is-style-faq-check-lists li::marker{
+	margin-right:10px;
+} 
+
+.is-style-faq-check-lists li { 
+	position: relative; 
+	padding-top: 12px;
+	padding-bottom: 12px;
+	padding-left: 30px;
+	border-top:1px solid rgba(255, 255, 255, 0.20);
+	border-bottom:1px solid rgba(255, 255, 255, 0.20);
+	font-size: var(--wp--preset--font-size--medium) !important;
+}
+
+.is-style-faq-check-lists li::before {
+    content: "\2193"; 
+    position: absolute;
+    left: 0;
```

Comment:
Another option would be using the [.rtl](https://github.com/WordPress/wordpress-develop/blob/6.3/src/wp-includes/post-template.php#L640) class that's automatically added to the body for RTL languages (if the idea is to avoid having a separate stylesheet).

Reply:
I recently tried to use the .rtl class in a Gutenberg block and found that it is not used in the iframed block editor(!)
The editor uses `<html dir="rtl"> ` and we can not target children of elements with this attribute.
See https://developer.mozilla.org/en-US/docs/Web/CSS/:dir

Reply:
Should we perhaps try to fix it in Gutenberg? https://github.com/WordPress/gutenberg/issues/51480

Reply:
I am not sure we have enough time.

Reply:
Should i try to use in different way  without children ? like as just user can copy the icon and list item will as normal list item .

Reply:
Then User will have to icon like this and space between the item like the example [https://prnt.sc/YnphTJPkKVnD] , What do you think ? as some of section in white background using this type pattern

Reply:
and one more thing, i tried to using marker and list style image, if i use list-style-position as normal , the list item border is not working as expected we want in design https://prnt.sc/avLaO4fbHqyR